lib/tokenize.cpp

```diff
@@ -8778,6 +8778,21 @@ void Tokenizer::simplifyKeyword()
                     }
                 }
             }
+
+            // noexcept -> noexcept(true)
+            // 3) void f() noexcept; -> void f() noexcept(true);
+            if (Token::Match(tok, ") noexcept [:{;]|const|override|final")) {
```

Comment:
Token::Match can't handle this syntax. Convert to:

    ) noexcept :|{|;|const|override|final